cmd/sealer/cmd/cluster/apply.go

```diff
@@ -127,12 +127,13 @@ func NewApplyCmd() *cobra.Command {
 					return applyClusterWithNew(cf, applyMode, imageEngine, imageSpec)
 				}
 
-				if err := applyClusterWithExisted(cf, client, imageEngine, imageSpec); err != nil {
+				updated, err := applyClusterWithExisted(cf, client, imageEngine, imageSpec)
```

Comment:
updated is too large to understand .  Or i thought should we move those run app logic  to applyClusterWithExisted.

Reply:
I have changed `updated` to `clusterUpdated`.

As with other comments, we should split run/apply to these steps:
1. distribute rootfs
2. apply/run cluster
3. app/run application

And then, something just like `clusterUpdated`, `skipPrepareAppMaterials ` will not be needed.

And we should do these code optimization ASAP.